### ISSUE TITLE
problem with testutils.py when importing odl, conebeam-branch

### ISSUE THREAD

**Issue by davlars** [odlgroup/odl#283]
When I try to import odl when on the issue-87__conebeam branch, a NameError is prompted for odl/util/testutils.py stating:

`File "odl/util/testutils.py", line 206, in <module>`
`     skip_if_no_cuda = _pass`
`NameError: name '_pass' is not defined`

I can though see that inside testutils.py `_pass` should be defined before line 206. Is this a known issue or only a problem locally for me?

**Comment by kohr-h** [comment]
This is a bug on the branch which has been fixed on master. The problem is that [here](https://github.com/odlgroup/odl/blob/745dd5163d77faaa93b887fc9c56b6473f5721fd/odl/util/testutils.py#L177-L209), the import error of `import pytest` is caught, but the `_pass` function is defined after that, so when the program flow reaches the `except` clause, the `_pass` function has never been defined.

The lazy workaround is to install `pytest`, the quick one to take `_pass` and move the code outside the `try..except` construct. The long-term, but unsatisfying solution is to wait for the branch to be updated to master.

**Comment by adler-j** [comment]
Easiest fix for you @davlars is to`pip install pytest`, we can close this once master is merged into conebeam.

**Comment by davlars** [comment]
That seem to have done the trick. Thanks.

**Comment by adler-j** [comment]
Has this been properly fixed now?

**Comment by kohr-h** [comment]
Hm, on the branch it's still unfixed. Probably since some change came later than the fix on master. I'll fix it, then we can close.
